src/Helper/Cron/Cancellation.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+/**
+ * Copyright Shopgate Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @author    Shopgate Inc, 804 Congress Ave, Austin, Texas 78701 <interfaces@shopgate.com>
+ * @copyright Shopgate Inc
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ */
+
+namespace Shopgate\Export\Helper\Cron;
+
+use Magento\ConfigurableProduct\Model\Product\Type\Configurable;
+use Magento\Framework\Message\ManagerInterface;
+use Magento\Sales\Model\Order as MagentoOrder;
+use Shopgate\Base\Api\Config\SgCoreInterface;
+use Shopgate\Base\Helper\Initializer\MerchantApi as MerchantApiHelper;
+use Shopgate\Base\Model\Shopgate\Order as ShopgateOrderModel;
+use Shopgate\Base\Model\Utility\SgLoggerInterface;
+use Shopgate\Export\Helper\Cron\Shipping as ShippingHelper;
+use Shopgate\Export\Helper\Cron\Utility as CronHelper;
+
+class Cancellation
+{
+    /** @var ManagerInterface */
+    private $messageManager;
+    /** @var SgLoggerInterface */
+    private $logger;
+    /** @var SgCoreInterface */
+    private $sgCoreConfig;
+    /** @var MerchantApiHelper */
+    private $merchantApiHelper;
+    /** @var \ShopgateMerchantApi */
+    private $merchantApi;
+    /** @var ShippingHelper */
+    private $shippingHelper;
+    /** @var CronHelper */
+    private $cronHelper;
+
+    /**
+     * @param ManagerInterface  $messageManager
+     * @param SgLoggerInterface $logger
+     * @param SgCoreInterface   $sgCoreConfig
+     * @param MerchantApiHelper $merchantApiHelper
+     * @param ShippingHelper    $shippingHelper
+     */
+    public function __construct(
+        ManagerInterface $messageManager,
+        SgLoggerInterface $logger,
+        SgCoreInterface $sgCoreConfig,
+        MerchantApiHelper $merchantApiHelper,
+        ShippingHelper $shippingHelper,
+        CronHelper $cronHelper
+    ) {
+        $this->messageManager    = $messageManager;
+        $this->logger            = $logger;
+        $this->sgCoreConfig      = $sgCoreConfig;
+        $this->shippingHelper    = $shippingHelper;
+        $this->merchantApiHelper = $merchantApiHelper;
```

Comment:
not even used, just a proxy for next line?

Reply:
yep, removed the property